tools/overlayhelpers/colliderinit.py

```diff
@@ -387,7 +387,7 @@ def GetColliderInit(address, type, num, path):
     else:
         return("ItemInit type must specify number of elements")
 
-    ovlFile = open(path + "/baserom/" + fileResult.name, "rb")
+    ovlFile = open(path + "/baseroms/" + fileResult.name, "rb")
```

Comment:
should be
```suggestion
    ovlFile = open(path + "/baseroms/gc-eu-mq-dbg/segments/" + fileResult.name, "rb")
```
afaict, but it's not like this script is going to be used again anyway (?)

same with other `tools/overlayhelpers/*.py`